prometheus_hardware_exporter/collector.py

```diff
@@ -5,12 +5,139 @@
 
 from prometheus_client.metrics_core import GaugeMetricFamily
 
+from .collectors.perccli import PercCLI
 from .collectors.storcli import StorCLI
 from .core import BlockingCollector, Payload, Specification
 
 logger = getLogger(__name__)
 
 
+class PowerEdgeRAIDCollector(BlockingCollector):
+    """Collector for PowerEdge RAID controller."""
+
+    perccli = PercCLI()
+
+    @property
+    def specifications(self) -> List[Specification]:
+        return [
+            Specification(
+                name="perccli_command_success",
+                documentation="Indicates if the command is successful or not",
+                labels=[],
```

Comment:
With the PR #4, the labels in Specification and Payload can be omitted if it's []. Same for other places

Reply:
Updated